application/models/meeting_fixtures_test.go

```diff
@@ -91,3 +92,37 @@ func createMeetingFixtures_FindByTime(ms *ModelSuite) Meetings {
 	}
 	return meetings
 }
+
+func createMeetingFixtures_CanUpdate(ms *ModelSuite) meetingFixtures {
```

Comment:
Since the UUT doesn't access the database, this test could just create in-memory fixtures. But having said that, I think my other suggestion will require database access.